<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFrequencies</class>
 <widget class="QMainWindow" name="AddFrequencies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddFrequencies</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>665</width>
      <height>239</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Frequency (Hz):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textEdit_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Defaults</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textBrowser"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="plotter_view"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
   </widget>
   <addaction name="menuLoad"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
